rosetta/failure/invalid_block.go

```diff
@@ -19,11 +19,11 @@ import (
 )
 
 type InvalidBlock struct {
-	Index   uint64
-	Hash    string
-	Message string
+	Description Description
+	Index       uint64
+	Hash        string
 }
 
 func (i InvalidBlock) Error() string {
-	return fmt.Sprintf("invalid block (index: %d, hash: %x): %s", i.Index, i.Hash, i.Message)
+	return fmt.Sprintf("invalid block (index: %d, hash: %x): %s", i.Index, i.Hash, i.Description)
```

Comment:
Since we have the block hash as a already hex-encoded string, should we not print it with `%s`?

e.g. - https://play.golang.org/p/gjHgKLIj1fP